core/src/macosX64Main/kotlin/Uuid.kt

```diff
@@ -7,4 +7,7 @@ import platform.Foundation.NSUUID
 
 internal fun Uuid.toNSUUID(): NSUUID = NSUUID(toString())
 internal fun Uuid.toCBUUID(): CBUUID = CBUUID.UUIDWithString(toString())
-internal fun CBUUID.toUuid(): Uuid = uuidFrom(UUIDString)
+internal fun CBUUID.toUuid(): Uuid = when (UUIDString.length) {
+    4 -> uuidFrom("0000$UUIDString-0000-1000-8000-00805F9B34FB")
```

Comment:
Is the additional string here ("000-1000-8000-008...") meaningful or is it just made up?

Reply:
Linked in the PR description, it's the base UUID for 16-bit Bluetooth UUIDs.

Reply:
Might be pertinent to add a comment explaining this?